tests/backup/backup_share_test.go

```diff
@@ -2527,16 +2519,13 @@ var _ = Describe("{ShareBackupsAndClusterWithUser}", func() {
 		err = DeleteBackupAndWait(userBackupName, ctxNonAdmin)
 		log.FailOnError(err, fmt.Sprintf("Failed while waiting for backup %s to be deleted", userBackupName))
 
-		backupUID, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
-		dash.VerifySafely(err, nil, fmt.Sprintf("Getting backup UID for backup %s", backupName))
-		_, err = DeleteBackup(backupName, backupUID, orgID, ctx)
-		dash.VerifyFatal(err, nil, fmt.Sprintf("Deleting backup - [%s]", backupName))
+		log.Infof("Deleting registered clusters for non-admin context")
```

Comment:
Should we put this in a loop to optimise it?

Reply:
Looping through what?

Reply:
We are not collecting the clusters in a list here

Reply:
I have not changed any logic here. I have just moved it from above to below. Earlier we were deleting the cluster even before the backups got deleted. This was causing issues for KDMP backups because KDMP backup deletion needs cluster reference and the cluster needs to be alive. So re-order it.
Now the Cluster is deleted only after the backups are deleted for non-admin context

Reply:
Okay No issues.